CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # 1.2.1
 
+## Dialog Full screen
```

Comment:
Usually we format release notes like this:

* `Dialog Full screen`: The `carbon-dialog-full-screen--open` class is now applied to the `html` element instead of the `body`.